OrbitClientServices/TracepointServiceClient.cpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OrbitClientServices/TracepointServiceClient.h"
+
+#include "OrbitBase/Logging.h"
+
+using orbit_grpc_protos::GetTracepointListRequest;
+using orbit_grpc_protos::GetTracepointListResponse;
+using orbit_grpc_protos::TracepointInfo;
+using orbit_grpc_protos::TracepointService;
+
+TracepointServiceClient::TracepointServiceClient(const std::shared_ptr<grpc::Channel>& channel)
+    : tracepoint_service_(TracepointService::NewStub(channel)) {}
+
+std::unique_ptr<grpc::ClientContext> TracepointServiceClient::CreateContext() const {
+  auto context = std::make_unique<grpc::ClientContext>();
+
+  return context;
+}
```

Comment:
nit: you can probably inline this there is only one function using it.

Reply:
This idea is great. Thank you so much! I wil.